src/server/cmd/pachd/main.go

```diff
@@ -90,6 +92,286 @@ func doReadinessCheck(config interface{}) error {
 	return env.GetPachClient(context.Background()).Health()
 }
 
+func doEnterpriseMode(config interface{}) (retErr error) {
```

Comment:
This is all copy-paste of `doFullMode` - at some point it seems like it would be good to DRY this up, but I think it's bigger surgery than I want to undertake in the 2.0 timeframe.